modules/wmco-prerequisites.adoc

```diff
@@ -7,27 +7,52 @@
 
 The following information details the supported platform versions, Windows Server versions, and networking configurations for the Windows Machine Config Operator. See the vSphere documentation for any information that is relevant to only that platform.
 
+[id="wmco-prerequisites-supported-5.1.0_{context}"]
+== WMCO 5.1.0 supported platforms and Windows Server versions
+
+The following table lists the link:https://docs.microsoft.com/en-us/windows/release-health/windows-server-release-info[Windows Server versions] that are supported by WMCO 5.1.0, based on the applicable platform. Windows Server versions not listed are not supported and attempting to use them will cause errors. To prevent these errors, use only an appropriate version for your platform.
+
+[cols="3,7",options="header"]
+|===
+|Platform
+|Supported Windows Server version
+
+|Amazon Web Services (AWS)
+|Windows Server 2019 (version 1809)
+
+|Microsoft Azure
+|Windows Server 2019 (version 1809)
+
+|VMware vSphere
+a|* Windows Server 2022 with the Windows link:https://support.microsoft.com/en-us/topic/april-25-2022-kb5012637-os-build-20348-681-preview-2233d69c-d4a5-4be9-8c24-04a450861a8d[KB5012637] patch.
+* Windows Server 20H2 
+
+|Bare metal or provider agnostic
+a|* Windows Server 2022 with the Windows link:https://support.microsoft.com/en-us/topic/april-25-2022-kb5012637-os-build-20348-681-preview-2233d69c-d4a5-4be9-8c24-04a450861a8d[KB5012637] patch.
+* Windows Server 20H2 
+|===
+
 [id="wmco-prerequisites-supported-5.0.0_{context}"]
 == WMCO 5.0.0 supported platforms and Windows Server versions
 
-The following table lists the link:https://docs.microsoft.com/en-us/windows/release-health/windows-server-release-info[Windows Server versions] that are supported by WMCO 5.0.0, based on the applicable platform. Windows Server versions not listed are not supported and attempting to use them will cause errors. To prevent these errors, use only an appropriate version for your platform.
+The following table lists the link:https://docs.microsoft.com/en-us/windows/release-health/windows-server-release-info[Windows Server versions] that are supported by WMCO 5.0.0, based on the applicable platform. Windows Server versions not listed are not supported and attempting to use them will cause errors. To prevent these errors, use only the appropriate version for your platform.
 
 [cols="3,7",options="header"]
 |===
 |Platform
 |Supported Windows Server version
 
 |Amazon Web Services (AWS)
-|Windows Server 2019
+|Windows Server 2019 (version 1809)
 
 |Microsoft Azure
-|Windows Server 2019
+|Windows Server 2019 (version 1809)
 
 |VMware vSphere
-|Windows Server 20H2 
+|* Windows Server 20H2 
 
 |Bare metal or provider agnostic
-|Windows Server 2019
+|* Windows Server 20H2 
```

Comment:
```suggestion
|Windows Server 2019 (version 1809) 
```
This should not be changed from 2019 to 20H2.
Remove `*` for consistency with other rows.